pageserver/src/tenant/mgr.rs

```diff
@@ -819,6 +813,11 @@ mod tests {
 
     #[tokio::test]
     async fn shutdown_joins_remove_tenant_from_memory() {
+        // the test is a bit ugly with the lockstep together with spawned tasks. the aim is to make
```

Comment:
> the aim is to make sure `shutdown_all_tenants0` per-tenant processing joins in any active remove_tenant_from_memory calls,

parser error. Is a verb missing?

Reply:
ah, the verb is "joins in".

Why not use "coalesces with"?

Reply:
Resolving in favor of the other thread: https://github.com/neondatabase/neon/pull/4678#discussion_r1267078921